mathics/builtin/numbers/calculus.py

```diff
@@ -1046,7 +1052,7 @@ def prepare_sympy(self, elements):
                 return [elements[0]] + x.elements
         return elements
 
-    def from_sympy(self, elements: Iterable) -> Expression:
+    def from_sympy(self, elements: list) -> Expression:
```

Comment:
"elements" we would prefer to be a tuple. Modifying the elements in place is generally not something we want to make easy. 

In the past, it was thought that this was done for "efficiency", but it just leads to subtle bugs.  We have also moved to nothing list expression properties (whether a list is already sorted, is a literal, can be represented in Python fully, etc).  Tuple help ensure these properties are up to date while lists will tend to allow for subtle mistakes.